recipes/multiping/meta.yaml

```diff
@@ -3,19 +3,19 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+{% set name = "multiping" %}
+{% set version = "1.1.12" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/5e/2d/58d80a082f1985c83db0c9927fccb3108cfc6c384d5c55b107a3066eea6b/multiping-1.1.2.tar.gz
```

Comment:
you def want the first form of the url.

Reply:
Couldn't get the first form to work, but was able to get a similar link to work based on the example link for Github.